packages/periphery/contracts/oracles/CurveOracle.sol

```diff
@@ -53,22 +67,26 @@ contract CurveEMAPriceOracle is IPriceOracle, Ownable2Step {
     uint8 baseDecimals = IERC20(baseToken).decimals();
     uint8 quoteDecimals = IERC20(quoteToken).decimals();
 
-    bool isForwardOrder = coin0 == quoteToken;
-    if (isForwardOrder && PRICE_DECIMALS + baseDecimals < quoteDecimals) {
+    bool isToken0Quote = coin0 == quoteToken;
+    if (isToken0Quote && PRICE_DECIMALS + baseDecimals < quoteDecimals) {
       revert ExtremeDecimals();
     }
-    if (!isForwardOrder && PRICE_DECIMALS + quoteDecimals < baseDecimals) {
+    if (!isToken0Quote && PRICE_DECIMALS + quoteDecimals < baseDecimals) {
       revert ExtremeDecimals();
     }
 
     OracleParams memory params = OracleParams({
       pool: pool,
-      isForwardOrder: isForwardOrder,
+      isToken0Quote: isToken0Quote,
+      priceOracleMethodHaveArg: priceOracleMethodHaveArg,
```

Comment:
Has*
I don't like this naming anyway however, need to think about the alternatives